httpclient5/src/main/java/org/apache/hc/client5/http/async/methods/SimpleHttpRequest.java

```diff
@@ -85,11 +85,11 @@ public void setBody(final SimpleBody body) {
         this.body = body;
     }
 
-    public void setBodyBytes(final byte[] bodyBytes, final ContentType contentType) {
+    public void setBody(final byte[] bodyBytes, final ContentType contentType) {
```

Comment:
@garydgregory Please keep the original methods. Just deprecate them.

Reply:
Done.